README.md

```diff
@@ -250,6 +250,31 @@ StartRun request:
 }
 ```
 
+## Using the Omics Run Analyzer tool
+### Basic Usage
+The `omics-run-analyzer` tool retrieves a workflow run manifest from CloudWatchLogs and generates statistics for the run, including CPU and memory utilization for each workflow task.
+
+#### List completed runs
+The following example lists all workflow runs completed in the past 5 days:
+```txt
+> omics-run-analyzer -t5d
+Workflow run IDs (<completionTime> <UUID>):
+1234567 (2024-02-01T12:00:00 12345678-1234-5678-9abc-123456789012)
+2345678 (2024-02-03T13:00:00 12345678-1234-5678-9abc-123456789012)
+```
+
+#### Analyze a specific workflow run
+```txt
+> omics-run-analyzer 1234567 -o run-1234567.csv
+omics-run-analyzer: wrote run-1234567.csv
+```
+
+#### Output workflow run manifest in JSON format
+```txt
+> omics-run-analyzer 1234567 -s -o run-1234567.json
```

Comment:
Can this be the default behavior? Additionally, having files written can be a pipe operation so:

```bash
# produces JSON output to STDOUT
omics-run-analyzer 123456

# capture output to file:
omics-run-analyzer 123456 > run-123456.json
```

Also, I think the script could benefit from a `-f|--format` option where the accepted values are `{(json),csv}` (json is default). The users can produce CSV output with:

```
omics-run-analyzer 123456 -f csv > run-123456.csv
```

Reply:
Yes, if you don't specify `-o <path>` output is written to stdout. In UNIX CLI commands, `-f` is typically reserved for an input file, as it is here in the current implementation, e.g.
```
omics-run-analyzer 1234567 -s -o run-1234567.json
omics-run-analyzer -f run-1234567.json -o run-1234567.csv
```